language/move-prover/tests/sources/regression/Choice_eliminate_reference.move

```diff
@@ -0,0 +1,11 @@
+address 0x1 {
+module Choice_eliminate_reference {
```

Comment:
I suggest to add this test to the existing `choice.move` test in sources/functional. It is obviously a test case missing there,